cmake/KokkosCore_config.h.in

```diff
@@ -26,6 +26,7 @@
 #cmakedefine KOKKOS_ENABLE_MEMKIND
 #cmakedefine KOKKOS_ENABLE_LIBRT
 #cmakedefine KOKKOS_ENABLE_SYCL
+#cmakedefine KOKKOS_SYCL_DEVICE_GLOBAL_SUPPORTED
```

Comment:
Should this guy prefixed with `IMPL_`

Reply:
Sure, in particular, we want to get rid of it when the compiler supports the feature macro.

Reply:
Please prefix and I will approve

Reply:
Here you go!